String changes to the SSO CILogon Sign-in button

200455939-yashu opened the pull request:
Fixes #903 .

Changes proposed in this PR: Changes to the SSO CILogon Sing-in button
-

200455939-yashu committed:
String changes to the SSO CILogon Sing-in button

aaronskiba reviewed (changes requested):
The main update is correct.

However, we also have tests that correspond with the name of this button. Please update these as well.

```
Failures:

  1) Openid_connection SSO with correct credentials creates account from external credentials
     Failure/Error: click_link 'Sign in with CILogon'

     Capybara::ElementNotFound:
       Unable to find link "Sign in with CILogon"
     # ./spec/integration/openid_connect_sso_spec.rb:29:in `block (3 levels) in <top (required)>'
```

aaronskiba committed:
Update test to correspond with update of SSO button

aaronskiba committed:
Set text as `.html_safe` to use `<br>` tag

The corresponding French translation of this text is too long to only be a single line long. To address that, `.html_safe` has been added here, which enables the correct behaviour when adding the `<br>` tag for the French translation on translation.io

lagoan reviewed (dismissed):
LGTM!

lagoan committed:
Update CHANGELOG
